spec/models/form_profile_spec.rb

```diff
@@ -866,6 +866,48 @@
     }
   end
 
+  let(:v21_22_expected_) do
```

Comment:
Delete the trailing `_` in `v21_22_expected_`.

Also, where is this being used?